builtin/providers/openstack/resource_openstack_compute_instance_v2.go

```diff
@@ -965,18 +965,19 @@ func getInstanceNetworks(computeClient *gophercloud.ServiceClient, d *schema.Res
 		allPages, err := tenantnetworks.List(computeClient).AllPages()
 		if err != nil {
 			if _, ok := err.(gophercloud.ErrDefault404); ok {
+				log.Println("[DEBUG] os-tenant-networks disabled")
 				tenantNetworkExt = false
 			}
 
+			log.Println("[DEBUG] Err looks like: %+v", err)
```

Comment:
@jtopjian This line is throwing a 'vet' violation: 
>builtin/providers/openstack/resource_openstack_compute_instance_v2.go:972: possible formatting directive in Println call

Want me to raise a PR changing "_Println_" to "_Printf_"?